### ISSUE TITLE
add presets for PL setting combinations, e.g. "allow only reactions"

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#20492]
I suggested in #20450:

> it might be useful to have some presets like "announcement room, users can only react" which sets message = mod, reaction = redact = default

saibotk commented:

> While #7492 added a settings switch, this still leads to weird UX, where Users can send reactions but are not allowed to remove them. (https://github.com/vector-im/element-web/issues/20450#event-5865321617)

**Comment by saibotk** [comment]
Wait, so is there an actual workaround using the devtools?
I did not figure out which power level i need to adjust to what value, so people can just remove their reactions.

**Comment by HarHarLinks** [comment]
No, you're right, `m.room.redaction` is needed and was missed in #7492
